.operations/CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Contribution model
+
+### Steering committee 🏆
+
+The steering committee work together to provide guidance and future direction to the project. Each committee member has a particular experitse which they share their knowledge on, and work to lead further improvements to the project in that are. In addition, each member of the committee leads a project which are tracked under Github projects.
```

Comment:
I'm not sure about the `work together` of the committee language-wise, is the word used in the sense of plural or wouldn't you rather use 'the steering committee works together'? I'd be more familiar with something like the latter or "Members of the steering committee work together..."